benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +174,64 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    pretty_name = "Fit Methods Overridden with Analytical MLEs"
+    param_names = ["distribution", "loc_fixed", "scale_fixed",
+                   "shape1_fixed", "shape2_fixed", "shape3_fixed"]
+    dists = ["pareto", "laplace", "beta", "ncf"]
+    shape1, shape2, shape3, loc, scale = [[True, False]] * 5
+
+    params = [dists, loc, scale, shape1, shape2, shape3]
+
+    distributions = {"pareto": {"self": stats.pareto, "floc": 0, "fscale": 2,
+                                "shape1_fixed": 2, "fixed_shapes": {"b": 2},
+                                "loc": 2, "scale": 3},
+                     "laplace": {"self": stats.laplace, "floc": 0,
+                                 "fscale": 2, "loc": 2, "scale": 3},
+                     "beta": {"self": stats.beta, "floc": 1,
+                              "fscale": 2, "loc": 1, "scale": 2,
+                              "fixed_shapes": {"b": 2.31, "a": 0.627},
+                              "shape1_fixed": 2.31, "shape2_fixed": .6},
+                     "ncf": {"self": stats.ncf, "floc": 0, "fscale": 2,
+                             "shape1_fixed": 2, "shape2_fixed": 3,
+                             "shape3_fixed": 2, "loc": 2, "scale": 3,
+                             "fixed_shapes": {"dfn": 1, "dfd": 2, "nc": 3}}
+                     }
```

Comment:
In the dictionary there is:
- self (the distribution)
- values to use for `fscale`, `floc`, and `shape1_fixed` `shape2_fixed`  `shape3_fixed` (if applicable)
- fixed_shapes: a dictionary that has the actual keyword names needed to generate data

Reply:
- Please rename "self". Maybe "class"? For clarity, "self" should be reserved for a reference to an instance of the class you are currently in. This is a handle to a separate class. 
- Please include this sort of information as a comment in the code
- Can we simplify this? Why not just, e.g. `f0`, `f1`, which can be used as keywords for any distribution's fit method (up to the number of shapes it has)?

Reply:
>  - Please rename "self". 

Removed the dictionary entirely, no longer relevant. 
 >   - Please include this information in the GitHub comment (about what this data means) as a comment in the code
>    - Can we simplify this? Why not just, e.g. f0, f1, which can be used as keywords for any distribution's fit method (up to the number of shapes it has)?
>   -  Or, even better, can we get the shape values directly from _distr_params.py and pick a default floc and fscale to use for every distribution so they don't need to be specified? It seems that to add a distribution to the benchmarks, all we should really need to enter here is the name, maybe the class (although there are probably ways around that), and guesses (if desired). If equally convenient, consider a flat list instead of a nested dictionary.
> 

Dictionary is removed, now the user only has to
- add a new distribution by adding it to the string list of distributions
- add rvs / fit values (if desired) by adding a flat list with the distribution name as the key in a dictionary. (Instructions are commented above it).